APMrover2/GCS_Mavlink.cpp

```diff
@@ -693,6 +693,13 @@ MAV_RESULT GCS_MAVLINK_Rover::handle_command_long_packet(const mavlink_command_l
         }
         return MAV_RESULT_ACCEPTED;
 
+    case MAV_CMD_DO_SET_REVERSE:
+        // param1 : Direction (0=Forward, 1=Reverse)
+        if(!rover.control_mode->set_reversed(packet.param1)) {
```

Comment:
this is perhaps not quite correct because it only allows the GCS to make the vehicle go in reverse.  It doesn't allow the GCS to recover from the reverse by sending a DO_SET_REVERSE with param1 = 0.